virttest/ssh_key.py

```diff
@@ -32,19 +32,19 @@ def get_public_key():
     has_rsa_keypair = (os.path.isfile(rsa_public_key_path) and
                        os.path.isfile(rsa_private_key_path))
 
-    if has_dsa_keypair:
-        print('DSA keypair found, using it')
-        public_key_path = dsa_public_key_path
-
-    elif has_rsa_keypair:
+    if has_rsa_keypair:
         print('RSA keypair found, using it')
         public_key_path = rsa_public_key_path
 
+    elif has_dsa_keypair:
+        print('DSA keypair found, using it')
```

Comment:
can you help to make use of logging instead here and elsewhere.

Reply:
Updated